### DISCUSSION TITLE
Update site needs to be "updated" ?

### DISCUSSION THREAD

**Discussion by karypid** [eclipse-pmd/eclipse-pmd#109] [Q&A] [answered]
In the releases page and all documentation, the update site used is: 

http://www.acanda.ch/eclipse-pmd/release/latest

This does not work for me (certificat validation error). 

I instead used the Eclipse Marketplace to install (which worked) and post-installation the update site added was:

https://eclipse-pmd.acanda.ch/release/latest

This seems to work fine as an update site from eclipse.

**Comment by acanda** [comment]
Thanks for reporting this. I had to change the update site URL a while ago when the Eclipse Marketplace added new requirements for the update sites. https://eclipse-pmd.acanda.ch/release/latest is the current update site that should work for both direct and marketplace installations.

The old URL still points to a valid update site, but it seems that Eclipse now requires https for direct installations as well. The releases page does have the old URLs and I need to change them. But I can't find any old URLs in the documentation. Can you provide links to documentation pages with the old URL?

**Reply by karypid** [reply]
Hello,

This is the top search result for me for "eclipse pmd download" (it's actually the third result, but the first two are for the other PMD plugin):

https://acanda.github.io/eclipse-pmd/getting-started.html

I realize now that this is from the pre-fork and therefore still has the old docs?

Perhaps a commit to the old repo telling users to refer to the new location would be useful? When visiting that page it was not apparent to me that I was reading old docs...

**Reply by acanda** [reply, marked as answer]
You are right, these are the old docs from the archived repository. I unpublished them to avoid any confusion. Thanks.
